scripts/build/VSImport-Samples.ps1

```diff
@@ -14,15 +15,18 @@ $ErrorActionPreference = "Stop"
 
 & $PSScriptRoot\unzip.ps1 -zipFile $zipFile -outputDir $tempDir
 
-$XCodeProjects = gci -recurse $tempDir\*.xcodeproj
+$XCodeProjects = gci -recurse $PSScriptRoot\..\..\samples\*.xcodeproj
+$XCodeProjects += gci -recurse $PSScriptRoot\..\..\tests\testapps\*.xcodeproj
```

Comment:
I think we should use the more generic thing where the xcode folder is passed in.